packages/suite/src/utils/suite/env.ts

```diff
@@ -27,7 +27,10 @@ export const submitRequestForm = async (
     if (formMethod === 'IFRAME') return;
 
     if (formMethod === 'GET' && formAction) {
-        window.open(formAction, isDesktop() ? '_blank' : formTarget);
+        setTimeout(() => {
+            // slightly delay opening of the new page, Suite Web may not complete storing data do DB otherwise
```

Comment:
type? "storing data to DB"?

Reply:
The browser is storing data to Index DB in the background (even though we are awaiting it :-( ) and aborts it if you open a new window too early.